packages/visx-xychart/src/hooks/useScales.ts

```diff
@@ -41,11 +41,18 @@ export default function useScales<
 
     const xDomain = isDiscreteScale(xScaleConfig) ? xValues : d3Extent(xValues);
 
-    let xScale = createScale({
-      range: [xMin, xMax],
-      domain: xDomain as [number, number],
-      ...xScaleConfig,
-    }) as XScale;
+    let xScale = (scaleCanBeZeroed(xScaleConfig)
+      ? createScale({
+          range: [xMin, xMax],
+          domain: xDomain as [XScaleInput, XScaleInput],
```

Comment:
noting that the domain was updated from `number` => `ScaleInput` (leftover from my comment [here](https://github.com/airbnb/visx/pull/987#discussion_r551586262))